test/common/testlib.py

```diff
@@ -854,6 +855,14 @@ def skipImage(reason, *args):
         return unittest.skip("{0}: {1}".format(testvm.DEFAULT_IMAGE, reason))
     return lambda func: func
 
+def skipPackage(*args):
+    packages_env = os.environ.get("TEST_SKIP_PACKAGES","").split()
+    for package in args:
+        if package in packages_env:
+            return unittest.skip("Not testing package (rule: {0} is in {1})".format(package, packages_env))
```

Comment:
Can you please clarify this to `"{0} is excluded in $TEST_SKIP_PACKAGES".format(package)`? This is enough and doesn't need to repeat the whole value of `$TEST_SKIP_PACKAGES` and doesn't need to talk about "rules".